pkg/deploy/internal/setting/validation_test.go

```diff
@@ -246,15 +247,50 @@ func TestInsertAfterSameScopeValidator(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "Valid reference to config in other project",
+			sourceConfig: config.Config{
+				Coordinate:  coordinate.Coordinate{Project: "project-id", Type: "type-a", ConfigId: "config-x"},
+				Environment: "env",
+				Type:        &config.SettingsType{SchemaId: "type-a"},
+				Parameters: map[string]parameter.Parameter{
+					config.InsertAfterParameter: refParam.New("other-project", "type-a", "config-y", "id"),
+					config.ScopeParameter:       valueParam.New("environment"),
+				},
+			},
+			otherProjectConfigs: []config.Config{
```

Comment:
Can you just add another config with the same ID and type, but from the 3rd project?

```
				{
					Coordinate:  coordinate.Coordinate{Project: "wrong-project", Type: "type-a", ConfigId: "config-y"},
					Environment: "env",
					Type:        &config.SettingsType{SchemaId: "type-a"},
					Parameters: map[string]parameter.Parameter{
						config.ScopeParameter: valueParam.New("environment"),
					},
				},
			},
```

Reply:
Did add the config